### PR TITLE
hardening/841_add_arraylist_to_string_function

### PR DESCRIPTION
**Pull request by pcoello25**
* Fix issue https://github.com/telefonicaid/fiware-cygnus/issues/841
* 100% unit test passed:
```
Results : Tests run: 88, Failures: 0, Errors: 0, Skipped: 0
```
* Assignee: @frbattid 

(Branch's name is `hardening/841_add_utils_to_string_function` but it should have been called `hardening/841_add_arraylist_to_string_function`.)

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -2,3 +2,4 @@
 - [BUG] Fix default authentication parameters in OrionMySQLSink and OrionPostgreSQLSink (#837)
 - [HARDENING] Use precompiled regexes in Utils.encode() and Utils.encodeHive() (#818)
 - [HARDENING] Remove a temporal fix regarding Orion <= 0.10.0 in OrionRESTHandler (#840) 
+- [HARDENING] Add an efficient function for converting ArrayList to Strings in Utils.java (#841)
```
Comment (frbattid) [inline CHANGES_NEXT_RELEASE]: to String

File: src/main/java/com/telefonica/iot/cygnus/utils/Utils.java [hunk 2]
```diff
@@ -72,10 +73,27 @@ public static String encodeHive(String in) {
     } // encodeHive
     
     /**
+     * Encodes a string from an ArrayList
+     * @param in
+     * @return The encoded string
+     */
+    public static String toString(ArrayList in) {
+        String out = "";
```
Comment (frbattid) [inline src/main/java/com/telefonica/iot/cygnus/utils/Utils.java]: Blank line require before and after the `for`block.

Comment (pcoello25) [inline src/main/java/com/telefonica/iot/cygnus/utils/Utils.java]: Fixed in bebf04b

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (pcoello25) [inline CHANGES_NEXT_RELEASE]: Fixed in bebf04b

**Comment by frbattid** [comment]
LGTM

**Pull request merged**